.circleci/config.yml

```diff
@@ -63,10 +63,7 @@ commands:
   run_test_bundlers:
     description: rerun unit tests on webpack and browserify bundled files
     steps:
-      - run: npm install browserify webpack webpack-cli
```

Comment:
The fix for the bug is to add `--no-save` here, as mentioned at https://github.com/mozilla/webextension-polyfill/issues/604#issuecomment-2110749236